### PR TITLE
Add config to run Ui Server in a non-VM environment

### PR DESCRIPTION
**Pull request by foochris**

### CODE REVIEW

**Comment by mcstover** [comment]
Hey, there might be another way to do this that doesn't end up with so much repetition... please hold for now.

File: config/dev-local.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+var merge = require('webpack-merge')
+var base = require('./index.js')
+
```
Comment (mcstover) [inline config/dev-local.js]: You can import dev-vm here too....

File: config/dev-local.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+module.exports = merge(base, {
+	app: {
```
Comment (mcstover) [inline config/dev-local.js]: then `merge(base, dev-vm, {....`

And you'd only need to include items that need altering...app { publicPath }, server { memcached } etc...

**Comment by foochris** [comment]
@mcstover PR updated.

**Comment by mcstover** [comment]
FYI @foochris I merged in the new apache configs earlier today so the proxy pass for /uiext should show up in the vms soon.

**Pull request merged**